app/src/main/java/bg/zahov/app/ui/workout/add/AddTemplateWorkoutFragment.kt

```diff
@@ -162,7 +163,7 @@ class AddTemplateWorkoutFragment : Fragment() {
             addExercise.setOnClickListener {
                 findNavController().navigate(
                     R.id.create_workout_template_to_add_exercise,
-                    bundleOf("SELECTABLE" to true)
+                    bundleOf("ADDABLE" to true)
```

Comment:
should also be a constant

Reply:
They will be changed once we change the navigation to compose